client-data/js/board.js

```diff
@@ -167,10 +171,34 @@ async function onCaptureClick() {
 }*/
 function onCaptureClick() {
 	console.log("capture clicked");
+	if(mode=="whiteboard"){
+		getWbSnapshot();
+	} else if (mode=="station"){
+		startPlainSnapshot();
+	}
+}
+
+function getWbSnapshot(){
 	toggleLoadingImg(true);
 	Tools.send({type:"robotmessage", msg:"getwbsnapshot"},"robotTool");
 }
 
+function startPlainSnapshot(){
+	
+	if (!window.confirm("Delete this board, and proceed to take snapshot from robot's camera?")) return;
+	
+	Tools.drawAndSend({'type': 'deleteall',},Tools.list.Clear);
+	Tools.send({type:"robotmessage", msg:"showblack"},"robotTool");
+	setTimeout(getPlainSnapshot, 250);
```

Comment:
I don't understand why this needs a delay. Please remove it or add a comment about why there is a delay.